documentation/docs/roadmap.md

```diff
@@ -37,7 +37,22 @@ The timeline below shows the past milestones we have completed, along with the f
 * Add support for 0x v3
 
 ## Q1 2020
-* Liquidity Mining Launch: Wave 1 (mid-February)
-* More connectors for decentralized and centralized exchanges: KuCoin, Bitfinex, HitBTC, gate.io
+* Liquidity Mining Launch: Open Beta (February 3, 2020)
+* Liquidity Mining Launch: Wave 1 (mid-March)
```

Comment:
early March

Reply:
"on Binance and 0x/Bamboo Relay"

Reply:
Done with requested changes in commit https://github.com/CoinAlpha/hummingbot/pull/1377/commits/796ac078d04a83b3114ac0d72a9990bf4cacd333.